### PR TITLE
Added my project pi_payments

### PR DESCRIPTION
**Pull request by anshulahuja98**
https://github.com/anshulahuja98/pi_payments
A payment module made on Raspberry based on RFID.

- [x] I have read and understood the [contribution guidelines](https://github.com/thibmaek/awesome-raspberrypi/blob/master/CONTRIBUTING.md).
- [x] This pull request has a descriptive title. *(For example: `Add Raspbian`)*
- The topic I added
	- [x] includes a valid (https) link,
	- [x] includes a concise and on-topic description,
	- [x] mentions if there is only support some devices,
	- [x] is not a duplicate,
	- [x] has been added at the bottom of the appropriate category,
	- [x] is in the form of **Named Link - Description, seperated by commas.**
	- [x] ends with a dot.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,6 +154,7 @@ The complete collection of Raspberry Pi models consist of:
 - [Occu-Pi](https://github.com/bww/occu-pi) - Controller software for the Occu-pi, a totally awesome bathroom door sensor.
 - [docsis-cable-load-monitor](https://github.com/sp4rkie/docsis-cable-load-monitor) - Tool to monitor downstream load on DOCSIS cable networks.
 - [BeeMonitor](https://beemonitor.org/setup/) - Honey beehive monitoring project.
+- [pi_payemnts](https://github.com/anshulahuja98/pi_payments) - A payment module made on Raspberry based on RFID.
```
[the remark this answers is not in the document]
Comment (anshulahuja98) [reply README.md]: Okay, I'll fix it.

**Comment by anshulahuja98** [comment]
Do you want me to squash it into one commit, or will you do that on merging?

**Comment by thibmaek** [comment]
Nope that's fine :)

**Comment by anshulahuja98** [comment]
Oh sorry about the typo, my bad, i didn't notice even after you mentioned

**Pull request merged**